kratos/utilities/divide_triangle_2d_3.cpp

```diff
@@ -214,8 +211,7 @@ namespace Kratos
                     // If the indexed keys is larger or equal to the number of nodes means that they are the auxiliar interface points
                     if ((node_i_key >= n_nodes) && (node_j_key >= n_nodes)) {
                         // Generate an indexed point line geometry pointer with the two interface nodes
-                        IndexedPointGeometryPointerType p_intersection_line = Kratos::make_shared<IndexedPointLineType>(mAuxPointsContainer(node_i_key),
-                                                                                                                       mAuxPointsContainer(node_j_key));
+                        IndexedPointGeometryPointerType p_intersection_line = GenerateLine(node_i_key, node_j_key);
```

Comment:
```suggestion
                        IndexedPointGeometryPointerType p_intersection_line = GenerateIntersectionLine(node_i_key, node_j_key);
```
To make the method more self-descriptive.